projects/ccf-rui/src/app/shared/components/opacity-slider/opacity-slider.component.scss

```diff
@@ -31,12 +31,13 @@
     .mat-slider-track-wrapper, .mat-slider-track-fill {
       height: .2rem;
     }
+    
     .mat-slider-track-fill {
       opacity: 0.5;
     }
 
     .mat-slider-thumb {
-      transform: none;
+      transform: none !important;
```

Comment:
@axdanbol any alternative to !important here? Otherwise, the PR looks good.

Reply:
Probably not since it needs to change the behavior of a material component

Reply:
I changed it because I noticed the thumb shifts slightly when you click it in Firefox. This was the only way I could get it to work consistently across browsers.